libbeat/docs/shared-geoip.asciidoc

```diff
@@ -0,0 +1,118 @@
+[id="{beatname_lc}-geoip"]
+== Export geoIP information
+
+You can use {beatname_uc} along with the {ref}/geoip-processor.html[geoIP
+processor] in {es} to export geographic location information based on client
+IPs. Then you can use this information to visualize the location of your clients
+on a map in {kib}.
+
+The geoIP processor adds information about the geographical location of
+IP addresses, based on data from the Maxmind GeoLite2 City Database. Because the
+processor uses a geoIP database that's installed on {es}, you don't need
+to install a geoIP database on the machines running {beatname_uc}.
+
+NOTE: If your use case involves using {ls}, you can use the
+{logstash-ref}/plugins-filters-geoip.html[GeoIP filter] available in {ls}
+instead of using the geoIP processor. However, using the geoIP processor is the
+simplest approach when you don't require the additional processing power of
+{ls}.
+
+[float]
+[id="{beatname_lc}-configuring-geoip"]
+=== Configure the geoIP processor
+
+To configure {beatname_uc} and the ingest geoIP processor:
+
+1. Define an ingest node pipeline that uses a `geoip` processor to add location
+information to the event. For example, you can use the Console in {kib} to
+create the following pipeline:
++
+--
+[source,json]
+-------------------------------------------------------------------------------
+PUT _ingest/pipeline/geoip-info
+{
+  "description": "Add geoip info",
+  "processors": [
+    {
+      "geoip": {
+        "field": "client.ip",
+        "target_field": "client.geo",
+        "ignore_failure": true
```

Comment:
Should this be ignore_missing? https://www.elastic.co/guide/en/elasticsearch/reference/master/geoip-processor.html